dojo/templates/dojo/custom_html_report_finding_list.html

```diff
@@ -19,7 +19,7 @@ <h4>{{ finding.severity|capfirst }}</h4>
             <div class="panel-heading finding-title">
                 <div class="clearfix">
                     <h5>
-                        Finding {{ forloop.counter }}: {{ finding.title }}
+                        Finding {{ forloop.counter }} - DefectDojo ID {{ finding.id }}
```

Comment:
Just realized the title of the finding is removed here, and not displayed any where else in the report. Was this intended?

Reply:
It was, due to the description covering the finding, but on second thoughts it might be important. I'll add it, but not in the main heading as it can be really bulky